cfme/tests/infrastructure/test_tenant_quota.py

```diff
@@ -316,3 +337,59 @@ def test_vm_migration_after_assigning_tenant_quota(appliance, small_vm, set_root
     migrate_request.wait_for_request(method='ui')
     msg = "Request failed with the message {}".format(migrate_request.row.last_message.text)
     assert migrate_request.is_succeeded(method='ui'), msg
+
+
+# Args of parametrize is the list of navigation parameters to order catalog item.
+# first arg of parametrize is the list of fixtures or parameters,
+# second arg is a list of lists, with each one a test is to be generated
+# sequence is important here
+# indirect is the list where we define which fixtures are to be passed values indirectly.
+@pytest.mark.parametrize('context', [ViaSSUI, ViaUI])
+@pytest.mark.parametrize(
+    ['set_roottenant_quota', 'custom_prov_data', 'extra_msg', 'set_default'],
+    [
+        [('cpu', '2'), {'hardware': {'num_sockets': '8'}}, '', True],
+        [('storage', '0.01'), {}, '', True],
+        [('memory', '2'), {'hardware': {'memory': '4096'}}, '', True],
+        [('vm', '1'), {'catalog': {'num_vms': '4'}}, '###', True],
+    ],
```

Comment:
Optional:
I think @ganeshhubale we can put this as global variable as `dict` and then parametrize.

Reply:
There are other test cases in other files too which needs same improvement. 
I will make new PR for it.
Thanks for suggesting